float64 move_angle
float64 move_speed
float64 move_turn_right
float64 move_turn_left
bool turn_right
bool turn_left
bool arm_data_1
bool arm_data_2
bool calibration
bool expansion_up
bool expansion_up_key
bool expansion_down
bool expansion_down_key
bool speed_half
bool loading
bool shooting
bool laundry_case_open
bool laundry_case_close
bool hand
